src/core/p5.Graphics.js

```diff
@@ -60,6 +60,13 @@ p5.Graphics = class extends p5.Element {
 
     if (r === constants.WEBGL) {
       this._renderer = new p5.RendererGL(this.canvas, this, false);
```

Comment:
I think we want this for both graphics and the main canvas, so maybe we should do the check inside RendererGL instead?